contracts/libraries/Math.sol

```diff
@@ -23,4 +23,16 @@ library Math {
         if (a < b) return a;
         return b;
     }
+
+    function add(uint256 x, uint256 y) internal pure returns (uint256 z) {
```

Comment:
Are these new functions being used?  I looked around briefly but didn't see it.

Reply:
Nope I just removed those after I pushed up I saw I had some stuff left over